src/go/k8s/tests/e2e/create-topic-with-client-auth/00-assert.yaml

```diff
@@ -42,6 +42,27 @@ status:
 
 ---
 
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: cluster-tls-admin-client
+status:
+  conditions:
+    - reason: Ready
+      status: "True"
+      type: Ready
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: cluster-tls-user-client
```

Comment:
Assertions for this certificate seem duplicated (see below). Is there a reason for that?

Reply:
good catch, this was supposed to be admin. Thanks!